<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KbModeEventMgr</class>
 <widget class="QWidget" name="KbModeEventMgr">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mode Event Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="modeLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>WM_CLASS[0]</string>
       </property>
       <property name="text">
        <string>Instance Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Other operations are not supported</string>
       </property>
       <item>
        <property name="text">
         <string>is</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>contains</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="classNameLineEdit">
       <property name="placeholderText">
        <string>Steam</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="isContainsCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>is</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>contains</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Other operations are not supported</string>
       </property>
       <item>
        <property name="text">
         <string>is</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>contains</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="instanceNameLineEdit">
       <property name="placeholderText">
        <string>Navigator</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>_NET_WM_NAME or WM_NAME</string>
       </property>
       <property name="text">
        <string>Window Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>_NET_WM_PID</string>
       </property>
       <property name="text">
        <string>Executable</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Other operations are not supported</string>
       </property>
       <item>
        <property name="text">
         <string>is</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>contains</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>WM_CLASS[1]</string>
       </property>
       <property name="text">
        <string>Class Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="windowTitleLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>SuperTuxKart</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="programLineEdit">
         <property name="placeholderText">
          <string>/usr/bin/xterm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="la">
     <property name="text">
      <string>Leave fields blank to disable them</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="clearbtn">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
